src/ucp/core/ucp_ep.h

```diff
@@ -479,12 +479,13 @@ typedef struct {
 
 
 enum {
-    UCP_WIREUP_SA_DATA_CM_ADDR = 2      /* Sockaddr client data contains address
-                                           for CM based wireup: there is only
-                                           iface and ep address of transport
-                                           lanes, remote device address is
-                                           provided by CM and has to be added to
-                                           unpacked UCP address locally. */
+    UCP_WIREUP_SA_DATA_AM_ONLY   = UCS_BIT(0),  /* Address has only AM lane information. */
+    UCP_WIREUP_SA_DATA_CM_ADDR   = UCS_BIT(1)   /* Sockaddr client data contains address
```

Comment:
changing this value breaks wire compatibility

Reply:
Yes, but as we discussed at wire compact meeting:

- If someone has now address > priv_data_length it won't work, since the server will try to instantiate connections on all the lanes. So probably there are no users with this scenario now, otherwise, they'll file a bug.
- Worker_uuid won't be included in the address by default unless the user explicitly sets flag `SEND_WORKER_UUID` on an endpoint.  For users that will use this functionality, they are aware of not mixing UCX versions (Spark users).
- So for users that are currently using UCX and their address fits into private data no breaking changes.

Reply:
I mean this line: https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_ep.c#L748
Changing the value of `UCP_WIREUP_SA_DATA_CM_ADDR ` will fail the check in `ucp_ep_create_server_accept`

Reply:
pls disregard, it was not the bit, but the value, so it did not change